src/forcing/ForcingsEngineDataProvider.cpp

```diff
@@ -0,0 +1,47 @@
+#include <forcing/ForcingsEngineDataProvider.hpp>
+#include <utilities/python/InterpreterUtil.hpp>
+
+#include <iomanip> // std::get_time
+
+namespace data_access {
+namespace detail {
+
+// Initialize instance storage
+ForcingsEngineStorage ForcingsEngineStorage::instances{};
+
+time_t parse_time(const std::string& time, const std::string& fmt)
+{
+    std::tm tm_ = {};
+    std::stringstream tmstr{time};
+    tmstr >> std::get_time(&tm_, fmt.c_str());
+
+    // Note: `timegm` is available for Linux and BSD (aka macOS) via time.h, but not Windows.
```

Comment:
Is this thus missing `#include <time.h>`?

Reply:
I think `<iomanip>` transitively includes `<ctime>`, but it's probably better to explicitly include that. I'll add this in.